kubecost/values.yaml

```diff
@@ -1227,18 +1227,6 @@ clusterController:
   #         "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
   #         "client_x509_cert_url": ""
   #       }
-  # # Actions can reference configuration at a global or cluster-specific level
-  # config:
```

Comment:
This config block shouldn't be needed in the cluster controller values IIRC?

Reply:
correct, this lives in kubecostProductConfigs, see https://github.com/kubecost/kubecost/blob/8c72e3774bf864bacbcbb37bc20e8c7e5d731f88/kubecost/values.yaml#L1346